docs/1.getting-started/6.data-fetching.md

```diff
@@ -8,63 +8,63 @@ Nuxt comes with two composables and a built-in library to perform data-fetching
 
 In a nutshell:
 
-- [`useFetch`](/docs/api/composables/use-fetch) is the most straightforward way to handle data fetching in a component setup function.
-- [`$fetch`](/docs/api/utils/dollarfetch) is great to make network requests based on user interaction.
-- [`useAsyncData`](/docs/api/composables/use-async-data), combined with `$fetch`, offers more fine-grained control.
+- [`$fetch`](/docs/api/utils/dollarfetch) is the simplest way to make a network request.
+- [`useFetch`](/docs/api/composables/use-fetch) is wrapper around `$fetch` that fetches data only once in [universal rendering](/docs/guide/concepts/rendering#universal-rendering).
+- [`useAsyncData`](/docs/api/composables/use-async-data) is similar to `useFetch` but offers more fine-grained control.
```

Comment:
How about switching those two ? 
Since `useFetch` is a "wrapper" around `$fetch` and `useLazyAsyncData`, it's probably easier to understand if we explain in this order: `$fetch` +  `useAsyncData` ->`useFetch`

Reply:
That change did cross my mind. Although you're right, in a new Nuxt dev's point of view, understanding `$fetch` and `useFetch` first is more fundamentally important. `useAsyncData` is a step towards more customisability which comes later.